common/src/main/java/com/wynntils/wc/model/Character.java

```diff
@@ -111,7 +111,7 @@ public static CharacterInfo parseCharacter(ItemStack itemStack, int slotNum) {
             List<String> lore = ItemUtils.getLore(itemStack);
 
             int level = 0;
-            ClassType classType = null;
+            String className = "";
```

Comment:
Why?

Reply:
Sorry, I should have explained this further. This is a bug fix for Character, which did not properly detect if a player were reskinned. The old code did `ClassType.isReskinned(classType.getName())` but that was just begging the question since `classType.getName()` always returned the non-reskinned name. So we needed to save the *actual* name as given in the tooltip. (As usual, I prefer to use "" instead of null for empty strings, unless I'm absolutely sure no NPEs are possible, and I want to signal some kind of error condition)